spec/rspec/matchers_spec.rb

```diff
@@ -24,6 +24,16 @@
       end
     end
   end
+
+  describe "#respond_to?" do
+    it "returns true for methods in example scope" do
+      expect(self.respond_to?(:be_happy)).to eq(true)
+    end
+
+    it "returns false for methods not defined in matcher or example scope" do
+      expect(self.respond_to?(:have_eyes_closed)).to eq(true)
```

Comment:
There's a discrepancy between the doc string ("returns false....") and the expectation (`to eq(true)`).  Seems like you should move this expectation into the other example, and come up with a new message to check here.